src/teamPage.js

```diff
@@ -1,20 +1,59 @@
 import React from "react";
 import "./teamPage.css";
-import member from "./surprised.jpg";
+import member1 from "./team-photos/Jasmine.jpg";
+import member2 from "./team-photos/Angel.jpg";
+import member3 from "./team-photos/Sam.jpg";
+import member4 from "./team-photos/Blandy.jpg";
+import member5 from "./team-photos/Nikhil.jpg";
+import member6 from "./team-photos/Darshpreet.jpg";
+import member7 from "./team-photos/Davinder.jpg";
 
 function TeamPage() {
```

Comment:
line 11 will change to `class TeamPage extends React.Component`

wrap return() inside render() as follows:
`render() {`
   `return(<div className="teamPage">`
`YOUR CONTENT HERE`
`</div>);`
`}`

before render() but inside the new class TeamPage define **Component state**, below is my example.
![image](https://user-images.githubusercontent.com/26355782/68460365-972cda80-01bc-11ea-820a-99f9de26b51a.png)

And then access the array in the return function with `this.state.teamMembers`